cfgov/jinja2/v1/landing-page-1/index.html

```diff
@@ -55,5 +56,14 @@
 {% block content_sidebar scoped -%}
     <aside>
         {# TODO: Add sidebar organisms. #}
+        {% set lipsumText = lipsum(1, false, 10, 15) | safe %}
```

Comment:
Mind moving this to L:15 so we don't accidentally get two of these when this PR and https://github.com/cfpb/cfgov-refresh/pull/1062/files#diff-5d6e03d4ccd0862cdc7f9ed2fe08cec5R15 go in?